crypto/rand/drbg_lib.c

```diff
@@ -97,10 +97,10 @@ static unsigned int rand_drbg_flags[3] = {
 };
 
 static unsigned int master_reseed_interval = MASTER_RESEED_INTERVAL;
-static unsigned int slave_reseed_interval  = SLAVE_RESEED_INTERVAL;
+static unsigned int normal_reseed_interval  = NORMAL_RESEED_INTERVAL;
```

Comment:
Please remove the extra space before the `=` to align the two equal signs.

Reply:
Okay.